### ISSUE TITLE
Remote source url incorrect with wildcard unset

### ISSUE THREAD

**Issue by RomanenkoStud** [digitalfortress-tech/typeahead-standalone#80]
### What happened?

If i want to use url callback function i dont need wildcard, since I recieve query as parameter

```js
source: {
    identifier: options.identifier ?? DEFAULT_ID,
    remote: {
        url: function (query) {
            return (`${fetchURL}?${new URLSearchParams({q: query.slice(0, 2)})}`)
        }, // fetch by first 2 letters
    }
},
```

For example, if the query is "apple", the request is made to "applehttps://myapi?q=ap".

Because of this code at typeahead-standalone

```js
fetchWrapper
      .get(
        (typeof remote.url === 'function' ? remote.url(frozenInput) : remote.url).replace(
          remote.wildcard || '',
          frozenInput
        ),
        remote?.requestOptions
      )
```

that replace empty string with input regardless from remote.url type.
So i still need to set wildcard that wouldn't be matched, untill issue is fixed

### Affected version

Latest

### Reproduction URL

https://codepen.io/ahmopkbo-the-reactor/pen/YzbaEZP

### How to reproduce?

1. Set remote source url without wildcard pattern
2. Leave wildcard option undefined

**Comment by niketpathak** [comment]
Hello @RomanenkoStud, Thank you for reporting the bug. It has been fixed in v5.1.0

**Comment by niketpathak** [comment]
Hello @RomanenkoStud, Thank you for reporting the bug. It has been fixed in [v5.1.0](https://www.npmjs.com/package/typeahead-standalone)
